src/components/User/Badges/BadgeCard.css

```diff
@@ -0,0 +1,60 @@
+.BadgeCard {
+  max-width: 185px;
+  height: 206px;
+  background: var(--white-900);
+  border: 1px solid var(--alpha-black-300);
+  box-shadow: 0 1px 8px var(--alpha-black-200);
+  border-radius: 8px;
+  flex: 1 0 calc(33.333% - 20px);
+  padding: 20px 22px 10px 22px;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  cursor: pointer;
+}
+
+.BadgeCard:hover {
+  box-shadow: 0 0 8px #736e7d;
+}
+
+.BadgeCard__Icon {
+  padding-bottom: 10px;
+}
+
+.BadgeCard__Icon > .Badge__Icon.Badge__Icon--primary {
```

Comment:
Can't you achieve this without modifying a css class from the Badge component from this file? Do you need to modify only the primary variant?